### PR TITLE
✨ Add system for culling fish and no longer flip dying fish

### PR DESCRIPTION
**Pull request by SamBWagner**
#31 
Add the dying mechanism

### CODE REVIEW

File: src/fish.rs [hunk 1]
```diff
@@ -75,6 +76,7 @@ struct FishBundle {
     state: FishState,
     variant: FishVariant,
     sprite_sheet: SpriteSheetBundle,
+    can_dy: CanDie,
```
Comment (tkapa) [inline src/fish.rs]: Je lie be lie peet rat goo my `can_dy`

File: src/fish.rs [hunk 2]
```diff
@@ -203,20 +207,41 @@ pub fn fish_movement(
...
+            &CanDie,
+        ),
+        With<Fish>,
+    >,
 ) {
-    for (mut fish, transform, mut direction) in &mut fish_query {
+    for (mut fish, transform, mut direction, can_die) in &mut fish_query {
         // Flip the thing when at edge
-        if transform.translation.x < -1800. {
+        if transform.translation.x < -1800. && !can_die.dying {
```
Comment (tkapa) [inline src/fish.rs:222]:
```suggestion
if  can_die.dying {
return;
}

        if transform.translation.x < -1800. {
```

**Review by tkapa — CHANGES_REQUESTED** [review]

File: src/fish.rs (hunk 2 above)
Comment (SamBWagner) [reply src/fish.rs]: done

File: src/fish.rs (hunk 1 above)
Comment (SamBWagner) [reply src/fish.rs:79]: done

**Review by tkapa — APPROVED** [review]

**Pull request merged**